### PR TITLE
Feat: ServicePage

### PR DESCRIPTION
**Pull request by Sandesh-Ghimire**
## Description

Service page added with 3 cards in it.

---

## Issue

- [x] Fixes - #23 
- [x] Partially fixes - #23

### CODE REVIEW

**Comment by Sandesh-Ghimire** [comment]
Remove the div from app.tsx it causing lots of issues while centering a div lol. @nirajgiriXD @sumanjha108155

File: tailwind.config.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ export default {
     extend: {
       textColor: {},
       backgroundColor: {},
+      backgroundImage: {
+        'hero-pattern': "url('/img/conference.jpg')",
+      }
```
Comment (nirajgiriXD) [inline tailwind.config.js:10]: No the correct way of doing it.

File: src/components/ServicePage/index.tsx
Comment (nirajgiriXD) [inline src/components/ServicePage/index.tsx:1]: This should be created as a component then later used in the page.

**Review by nirajgiriXD — APPROVED** [review]
## General Feedback:

- The images are all of different type (`jpg`, `jpeg`, `webp`) and `size`. It would be better if all the images were similar in nature (`extension` and `size`).

**Pull request merged**